src/components/PlayerComponent.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+import { StyleSheet, Text, TouchableOpacity, View } from 'react-native';
+import PlayerLifeButton from './PlayerLifeButton';
+
+export class PlayerComponent extends Component {
+  state = {
+    currentLife: 40,
+    minLife: 0
```

Comment:
Same thing here, maybe make this `40` a constant.

Reply:
I'm fine with this being declared here as it is a state object